tools/release-guide

```diff
@@ -89,16 +89,23 @@ prepare()
 
     cd ..
 
+    trace "Building the docs"
+    mkdir build
+    cd build
+    "../extract/${subdir}/configure"
+    make "$DIRECTORY/html"
+    cd ..
```

Comment:
This feels wasteful -- we already build git during the release process, and thus at this point cwd is a configured tree (as at least `release-source` calls `make dist`). Are you concerned about being able to do these steps separately? If so, we need to add a doc build to cockpit's distcheck, to make sure it works out of a tarball, instead of just a git clone (doing that is a good idea anyway, of course)

Reply:
As per IRC discussion, testing this `make "$DIRECTORY/html"` during unit tests can be a follow-up.